models/FeedbackModel.ts

```diff
@@ -0,0 +1,41 @@
+import { BaseEntity, Column, Entity, Index, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { FeedbackType, PublicFeedback, Uuid } from '../types';
+import { UserModel } from './UserModel';
+
+@Entity('Feedback')
+export class FeedbackModel extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  uuid: Uuid;
+
+  @ManyToOne((type) => UserModel, (user) => user.feedback, { onDelete: 'CASCADE' })
+  @JoinColumn({ name: 'user' })
+  @Index('user_feedback_by_user_index')
+  user: UserModel;
+
+  @Column('varchar')
```

Comment:
should we specify column types in the rest of the `*Model.ts` files? models like EventModel don't have column types for strings

Reply:
I think right now it might be that `text` is specified and `varchar` is the default. Your call if you want to leave it as that or make all string types explicit, but might be good to.